web/src/features/map/Map.tsx

```diff
@@ -17,6 +17,7 @@ import {
   productionConsumptionAtom,
```

Comment:
I feel like this new logic could really benefit from some Cypress tests if possible, as it's hard to test this locally and this areas has previously been something where we easily made mistakes due to the complexity :)

Reply:
After many attempts getting tests to work the same on dev and on github workflows I've created a new cypress component test to validate the centering on location works